rbd/pool_nautilus.go

```diff
@@ -99,3 +99,30 @@ func PoolInit(ioctx *rados.IOContext, force bool) error {
 	ret := C.rbd_pool_init(cephIoctx(ioctx), C.bool(force))
 	return getError(ret)
 }
+
+// poolStats represents RBD pool stats variable.
+type poolStats struct {
+	stats C.rbd_pool_stats_t
+}
+
+// poolStatsCreate creates a new poolStats struct.
+//
+// Implements:
+//  void rbd_pool_stats_create(rbd_pool_stats_t *stats)
+func poolStatsCreate() *poolStats {
```

Comment:
Since these are unexported functions we could just return the C types now. However it's not a big deal at all.